internal/wasm/jit/jit_impl_amd64.go

```diff
@@ -102,6 +102,13 @@ var (
 	}
 )
 
+var (
+	// amd64CallingConvensionModuleInstanceAddressRegister holds *wasm.ModuleInstance of the
+	// next executing function instance. The value is set and used when making function calls
+	// or function returns in the ModuleContextInitialization. See compileModuleContextInitialization.
+	amd64CallingConvensionModuleInstanceAddressRegister = amd64.REG_R12
```

Comment:
don't forget the doc ref
```suggestion
	amd64CallingConventionModuleInstanceAddressRegister = amd64.REG_R12
```